app/models/shipit/commit.rb

```diff
@@ -219,7 +219,7 @@ def detach_children!
       children.detach!
     end
 
-    def pull_request?
+    def merge_request?
```

Comment:
I think this shouldn't be changed. The meaning of this method is to tell wether the commit looks like a Pull Request merge commit.

Reply:
Do you think the name `Commit#pull_request_merge?` would ever-so-slightly elevate that intent to the reader over `Commit#pull_request?`? Would `Commit#pull_request_merge?` be an acceptable name. Do you think it improves clarity at all for the reader? Or would you prefer we stick with `Commit#pull_request?` predicate?

Reply:
I think it's slightly too verbose. I'd rather stick to `pull_request?` for now, we can always rename it later on.

Reply:
Resolved in: https://github.com/Shopify/shipit-engine/pull/1095/commits/ce8120a5f1e10b478c6b0facf5893a2027974d60